Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,95 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using Console = System.Console;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
         {
+            string name = PromptString("name");
```

Comment:
This block of code in main could be moved to a function:
PromptPersonInfo or similar.

Reply:
I am unclear on this. Isn't Prompt String the function? I am calling the function PromptString there as far as I understand.